### PR TITLE
Improve focus styling

### PR DESCRIPTION
**Pull request by edwardhorsford**
This pr addresses some of the focus styling issues identified in our internal accessibility bash

## Selected filters tags unclear

The tags aren't fully compliant with GDS focus styles - they accidentally leave on the default focus outline. If that's removed, they then rely on colour alone to distinguish focus. I've used a box shadow similar to GDS' link focus styles to solve this.

After:
Screenshot 2021-02-18 at 12 48 26

Before:
Screenshot 2021-02-18 at 12 51 18

## Focus on active tabs

Active tabs rely on colour alone to indicate focus. Now the 'active' bar gets a bit bigger. This is a bit different than how the design system does it, but their markup is rather different and I was trying for a simple fix.

After:
header-focus

Before:
Screenshot 2021-02-18 at 12 52 50

Design system:
Screenshot 2021-02-18 at 12 52 10

## How to test

Focus the following items in turn - each item should be distinguishable as focused without relying on a change in colour alone - eg it gets a border or something gets thicker.

Items to check:
* Header navigation
* Tab navigation on records
* Selected filters

### CODE REVIEW

**Comment by edwardhorsford** [comment]
[Improve focus styling](https://trello.com/c/fNALBQ3L/1101-improve-focus-styling)

File: app/components/tab_navigation/style.scss [hunk 1]
```diff
@@ -85,6 +85,10 @@
   position: relative;
   text-decoration: none;
 
+  &:focus:before {
+    height: 7px; // Bar needs to change thickness so that we don’t rely on colour alone.
```
Comment (jesseyuen) [inline app/components/tab_navigation/style.scss:89]: Consider setting height at 8px here so it's more noticeable.

File: app/components/navigation_bar/style.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.moj-primary-navigation__link[aria-current]:focus:before {
+  height: 7px; // Make bar thicker so we don’t rely on colour alone to indicate focus
```
Comment (jesseyuen) [inline app/components/navigation_bar/style.scss:2]: Consider setting height at 8px here so it's more noticeable.

**Review by jesseyuen — APPROVED** [review]
Approved with comments left about the focus state on the nav items

File: app/components/tab_navigation/style.scss (hunk 1 above)
Comment (edwardhorsford) [reply app/components/tab_navigation/style.scss:89]: Think I'll leave as is for now - you can see it change size, and the *current* tab is probably the least important. Manage has also now put in a PR using these styles.

**Review by AbigailMcP — APPROVED** [review]

**Pull request merged**